netbox/core/plugins.py

```diff
@@ -91,15 +91,15 @@ def get_local_plugins(plugins=None):
             tag_line=plugin_config.description,
             description_short=plugin_config.description,
             is_local=True,
-            is_installed=True,
+            is_installed=plugin_name in registry['plugins']['installed'],
             installed_version=installed_version,
         )
 
     # Update catalog entries for local plugins, or add them to the list if not listed
     for k, v in local_plugins.items():
         if k in plugins:
             plugins[k].is_local = True
-            plugins[k].is_installed = True
+            plugins[k].is_installed = k in registry['plugins']['installed']
```

Comment:
I think we can copy this over directly?
```suggestion
            plugins[k].is_installed = v.is_installed
```